### DISCUSSION TITLE
Possible balance issue with Traceable mod using custom approach circles.

### DISCUSSION THREAD

**Discussion by CloudKeyzz** [ppy/osu#28585] [Ideas]
With the traceable mod now being ranked, I wanted to try and see how this would fare against other mods. It came to me that traceable being able to use custom approach circles might cause the mod to be played in a much easier than intended way. Being a skinner I wanted to test something to see how much easier a custom approach circle could make the mod.

I went and made the custom approach circle that would essentially use a normal hitcircle inside of the area that would normally be bigger in normal approach circles. It doesn't feel like this is the way the mod should be used and can't really be compared to traceable with a normal skin. 

Here is a video showing the custom approach circle with the mod on. 

https://github.com/ppy/osu/assets/173797039/50979295-27a9-4fdd-88c4-42739bff89cc

With the mod being ranked I just wanted this to be known for future adjustments to the mod going forward. 
Below is the file for the approach circle used.  
[approachcircle@2x.zip](https://github.com/user-attachments/files/15970398/approachcircle%402x.zip)

**Comment by peppy** [comment]
traceable could probably force the approach circle to a default skin version i guess

**Reply by cl8n** [reply]
that sounds fine to me -- but if possible, I think it'd be cool to just mask out a circle in the center of the approach circle, that way you can still use most "normal" skins without allowing it to be exploited like this

**Reply by peppy** [reply]
Yeah that's not going to happen.

To make this work as per my proposal, we'll need a way for a mod to force default skin elements. Which sounds like a nightmare on top of all the existing skin layers we have, so don't know if it will ever happen.
